config/example/.sp_magi.env

```diff
@@ -22,3 +22,4 @@ DEVNET=true
 SEQUENCER=true
 SEQUENCER_MAX_SAFE_LAG=200
 SEQUENCER_PK_FILE=./sequencer_pk.txt
+WATCHER_DELAY=4000
```

Comment:
nit: add a comment above this (this makes it look like it's a required param for sequencing)